sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -440,6 +440,11 @@ public partial class BlobCopyInfo
         public System.DateTimeOffset LastModified { get { throw null; } }
         public string VersionId { get { throw null; } }
     }
+    public enum BlobCopySourceTags
```

Comment:
Maybe make sure to merge the other Copy Source Tags PR first before this one? Though I don't think this would be that bad of a merge conflict.

Reply:
Yep!  This PR need to merge 2nd.  This is an artifact of merged STG 82 swagger.